docs/admin/access.rst

```diff
@@ -285,10 +285,10 @@ Custom access control
 ---------------------
 
 By choosing :guilabel:`Custom` as :guilabel:`Access control`, Weblate will stop
-managing access for a given project, and you can set up custom rules in the Django
+All users and groups can be managed using the Django admin interface,
```

Comment:
"Weblate will stop managing access for a given project" seems removed without any replacement (and the first part of the sentence is now unfinished).